### PR TITLE
angelica/atqr-126-create-notifications-enums

### PR DESCRIPTION
**Pull request by angeliralves**
Add the file with the Notification Enums.

### CODE REVIEW

File: libs/atqr/domain/src/lib/services/notificationsEnums.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export enum MessageTemplateId {
+  Challenge = 'Challenge',
+  YourOfficiationHaveBeenRequested = 'YourOfficiationHaveBeenRequested',
+  YourOfficiationRequestHaveBeen = 'YourOfficiationRequestHaveBeen',
+  YouHaveBeenChallenged = 'YouHaveBeenChallenged',
+  YourContender = 'YourContender',
+  TimeToOfficiateArrived = 'TimeToOfficiateArrived',
+  YourJudgeDidNotAnswered = 'YourJudgeDidNotAnswered',
+}
```
Comment (svallory) [inline libs/atqr/domain/src/lib/services/notificationsEnums.ts]: There are a LOT of missing items here. When you see these in Disl:

```
Challenge...
  ~Created
  ~Canceler
```

You need to create the items:

```
ChallengeCreated = 'ChallengeCreated',
ChallengeCanceled = 'ChallengeCanceled'
```

File: libs/atqr/domain/src/lib/services/notificationsEnums.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export enum YourContender {
+  AcceptedTheChallenge = 'AcceptedTheChallenge',
+  RejectedTheChallenge = 'RejectedTheChallenge',
+  Ignored = 'Ignored',
+  RemovedYouFromJudgeRole = 'RemovedYouFromJudgeRole',
+  CanceledTheChallenge = 'CanceledTheChallenge',
+  WereTheySuccessful = 'WereTheySuccessful',
+}
+
+
```
Comment (svallory) [inline libs/atqr/domain/src/lib/services/notificationsEnums.ts]: Enums in Disl always start with the `enum` keyword. These were not new enums but items of `MessageTemplateId` named as I explained above

**Review by svallory — DISMISSED** [review]

File: libs/atqr/domain/src/lib/services/notificationsEnums.ts (hunk 2 above)
Comment (angeliralves) [reply libs/atqr/domain/src/lib/services/notificationsEnums.ts]: Fix the bugs pointed out

Comment (yfernandes) [reply libs/atqr/domain/src/lib/services/notificationsEnums.ts]: This conversation is no longer valid as this code has been refactored. Resolving conversation.

**Review by albnunes — COMMENTED** [review]
Everything looks correct. But DeepScan has one issue

**Comment by yfernandes** [comment]
DeepScan Issue is valid however is out of the scope of this task. It only came up because the file was moved. I'll open an issue to fix what has been brought by DeepScan.

**Review by yfernandes — APPROVED** [review]
Didn't see anything major except what's reported by DeepScan which I'll open an issue for!

**Pull request merged**